src/internal/tokio.rs

```diff
@@ -6,7 +6,7 @@ where
     F: Future<Output = T> + Send + 'static,
     T: Send + 'static,
 {
-    tokio::task::Builder::new().name(&*format!("serenity::{}", name)).spawn(future)
+    tokio::task::Builder::new().name(&*format!("serenity::{}", name)).spawn(future).unwrap()
```

Comment:
You should either decorate this with `#[allow(clippy::unwrap_used)]`, or use `.expect()`.

Reply:
Probably `.expect("called outside tokio runtime")`. That way, readers of the code know it's very reasonable to panic here